src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -647,6 +666,48 @@ private void LaunchOnlineHelp(Uri uriToLaunch)
             }
         }
 
+#if !UNIX
+        /// <summary>
+        /// Gets the path to the default browser by querying the Windows registry.
+        /// </summary>
+        /// <returns></returns>
+        private string GetDefaultWebBrowser()
+        {
+            // Check if there is a default browser in the system.
+            const string httpRegkey = @"HKEY_CURRENT_USER\Software\Microsoft\Windows\Shell\Associations\UrlAssociations\http\UserChoice";
+            object progId = Registry.GetValue(httpRegkey, "ProgId", string.Empty);
+            if (progId != null && !string.Empty.Equals(progId))
+            {
+                // Query the registry to find the web browser path.
+                using (RegistryKey browserRegKey = Registry.ClassesRoot.OpenSubKey(progId + "\\shell\\open\\command", false))
+                {
+                    if (browserRegKey != null)
+                    {
+                        const string exeExtension = ".exe";
+                        string browserPath = browserRegKey.GetValue(null).ToString().Replace(/* remove the quotes */ "\"", "");
+                        if (browserPath != null)
```

Comment:
there are 2 issues in line 687 and 788:
1. Possible `NullReferenceException` at `browserRegKey.GetValue(null).ToString()` when `GetValue(null)` returns `null`.
2. If line 687 runs fine, then `browserPath` must not be null, because `Replace(...)` never returns null.

This is a chance to use the new C# 6.0 feature *null-conditional operator* '**?.**' to make the code concise, like:
```
using (RegistryKey browserRegKey = Registry.ClassesRoot.OpenSubKey(progId + @"\shell\open\command", false))
{
    const string exeExtension = ".exe";
    string browserPath = browserRegKey?.GetValue(null)?.ToString().Replace(/* remove the quotes */ "\"", "");
    if (!string.IsNullOrEmpty(browserPath))
    { ...
```
This could reduce the brackets needed for this method.